src/Umbraco.Web.UI.Client/src/views/common/overlays/mediaPicker/mediapicker.controller.js

```diff
@@ -133,8 +133,7 @@ angular.module("umbraco")
                     });
                 $scope.currentFolder = folder;
 
-                // for some reason i cannot set cookies with cookieStore
-                document.cookie = "umbLastOpenedMediaNodeId=" + folder.id;
+                localStorageService.set("umbLastOpenedMediaNodeId", folder.id);

```

Comment:
OK update on this I can reproduce & test this works OK.
With the test being done with a GUID for this PR stored in `LocalStorage`

It makes a request to 
`/umbraco/backoffice/UmbracoApi/Media/GetChildren?id=d249e476-00d3-4669-9131-47cebb2d0f22&pageNumber=0&pageSize=0&orderBy=SortOrder&orderDirection=Ascending&orderBySystemField=true&filter=`

Which does not return any results for the editor/user who does not have access to that media folder, but the JSON does return a little bit of data but nothing that is too sensitive.

```
{"pageNumber":0,"pageSize":0,"totalPages":1,"totalItems":2,"items":[]}
```

So @Shazwazza & @perploug this seems all good to me in that regards.